docs/en/week10/10-3.md

```diff
@@ -0,0 +1,85 @@
+---
+lang-ref: ch.10-3
+lecturer: Alfredo Canziani
+title: Transformer  Encoder-predictor-decoder architecture
+authors: Rahul Ahuja, jingshuai jiang 
+date: 15th April 2021
+---
+
+
+## The Transformer
+
+Before elaborating the encoder-predictor-decoder architecture, we are going to review two models we've seen before.
+
+
+### Conditional EBM latent variable architecture
+
+
+We should be familiar with the terminology of these modules from the previous lectures. 
+In the conditional EBM latent variable architecture, we have $\vect{x}$ the conditional variable which goes into a predictor. We have $\vect{y}$ which is the target value. The decoder modules will produce $\tilde{\vect{y}}$ when fed with a latent variable $\vect{z}$ and the output of the predictor. $E$ is the energy function which minimizes the energy between $\tilde{\vect{y}}$ and $\vect{y}$. 
+
+
+<center>
+<img src="{{site.baseurl}}/images/week10/10-3/ebm.png" style="zoom: 80%; background-color:#DCDCDC;" /><br>
+<b>Figure 1:</b> Conditional EBM Model.
```

Comment:
These are titles. Please, add a caption for this and all the other figures.

Reply:
fixed

Reply:
Wait, just to clarify, you mean we should write a caption explanation (Like few more words ) explaining the figures right? So for example instead of "Conditional EBM Model", "The architecture of a Conditional EBM Model."